cocotb/share/makefiles/Makefile.inc

```
@@ -150,6 +150,20 @@ endif
 
 export LIBPYTHON_LOC
 
+define check_vhdl_sources
+if [ "$(VHDL_SOURCES_$(LIB))" == "" ]; then \
+  >&2 echo "ERROR: VHDL_SOURCES_$(LIB) empty or undefined, while present in VHDL_LIB_ORDER."; \
```

```suggestion
  >&2 echo "ERROR: VHDL_SOURCES_$(LIB) is empty or undefined while present in VHDL_LIB_ORDER."; \
```
(not sure about the comma, not a native speaker either :-))

---

Comma should stay, `while` should be changed to `but`: `"ERROR: VHDL_SOURCES_$(LIB) is empty or undefined, but present in VHDL_LIB_ORDER."`

---

I setteld on:

```
>&2 echo "ERROR: VHDL_SOURCES_$(LIB) is empty or undefined, but '$(LIB)' present in VHDL_LIB_ORDER."; \
```

to clarify that `$(LIB)` should be present in `VHDL_LIB_ORDER`, not `VHDL_SOURCES_$(LIB)`. Agreed?